pages/casestudies/[id].js

```diff
@@ -10,14 +10,14 @@ import GenericLayout from "../../components/layout/GenericLayout";
 import CaseTOC from "../../components/CaseTOC";
 import { generateCaseStudyContent } from "../../lib/staticHelpers";
 
-const renderContent = (content, allComponents, level) => {
+const renderContent = (content, allComponents, level, isAgain=false) => {
```

Comment:
what is isAgain doing here??

Reply:
so basically `isAgain` checks if the renderContent function is called for second time. if it's called second time then it doesn't add additional indentation. (i.e., add `w-11/12 mx-auto` classess)